docs/reference/src/introduction/supported_platforms.md

```diff
@@ -0,0 +1,50 @@
+# Supported Platforms
+
+Slint runs on many desktop and embedded platforms and micro-controllers.
+
+The platform descriptions below cover what has been tested for deployment. For the development environment,
+we recommend using a recent desktop operating system and a recent compiler.
+
+Please contact [SixtyFPS GmbH](https://slint.dev/contact) if you need to support specific, older versions.
+
+## Desktop Platforms
+
+Generally, Slint runs on Windows, macOS, and popular Linux distributions. The following tables below
+cover versions that we specifically test. The general objective is to support the operating systems that
+are supported by their vendors by the time a Slint version is released.
+
+### Windows
+
+| Operating System | Architecture |
+|------------------|--------------|
+| Windows 10       | x86-64       |
```

Comment:
Do we test Windows 10?  What does that mean that we test it?

Reply:
We run our tests on Windows Server 2022, which is the Windows 10 code base AFAIK.